### PR TITLE
welcome: Prevent soft-fail reference to fixed bug

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

**Comment by OleksandrOrlov** [comment]
LGTM, as the appropriate ticket in bsc is 'VERIFIED FIXED'.

Just my thoughts, but it would be great if in the appropriate bsc ticket you'll place a link to the passed build.

Also, verification run will not be superfluous here.

**Comment by okurz** [comment]
> Just my thoughts, but it would be great if in the appropriate bsc ticket you'll place a link to the passed build.

Normally we do this but in this case the resolution did not result in the simple case of failed->passed job, also see https://bugzilla.suse.com/show_bug.cgi?id=1056413#c18 and the surrounding comments for more details

> Also, verification run will not be superfluous here.

https://travis-ci.org/os-autoinst/os-autoinst-distri-opensuse/builds/413919842?utm_source=github_status&utm_medium=notification is the verification run. Do you mean we should need an openQA job to show that removing a test API call that has no impact on the test flow would change something?

**Comment by OleksandrOrlov** [comment]
Sorry if I look picky, but I'm doing the review of a PR for the first time:)

And what I noticed here as a person, who are not in the context of the issue, is that neither commit message, nor commit body and PR comment (that do not exist), do not give me a proper understanding of **why** you did the change, like removing record_soft_failure, but left 'if match_has_tag(...)' line.

Do you think, that reviewer should read tons of comments before making a decision upon a PR instead of just having a brief description of why the PR was made?

And the last thing that finally confused me is the comment in the code itself:

>             # assuming the according needle is handling the potential
>             # soft-fail and bug ref we can just continue here

What that comment will say to a person who will look at that part of code in 1 year and is not in the context of the issue and even do not know that soft-fail was here some time ago?

**Comment by asmorodskyi** [comment]
1. agree that no need for proof run 
2. commit message looks good enough for me 
3. agree with @OleksandrOrlov  - you wrote a comment from perspective of this change, better write  comment which would bring some info about code state AFTER your change and not why you remove soft-failure or just remove it.

File: tests/installation/welcome.pm [hunk 1]
```diff
@@ -65,7 +65,8 @@ sub run {
             next;
         }
         if (match_has_tag 'no-product-found-on-scc') {
-            record_soft_failure 'bsc#1056413';
+            # assuming the according needle is handling the potential
+            # soft-fail and bug ref we can just continue here
             wait_screen_change { send_key $cmd{ok} };
```
Comment (rwx788) [inline tests/installation/welcome.pm]: Do we have a test which clicks `Next` without selecting product? I would remove whole if branch, as we will hide an error if this pop-up is shown.

**Comment by foursixnine** [comment]
@okurz do you still find this relevant?.

**Comment by okurz** [comment]
Yes

**Comment by rwx788** [comment]
@okurz if you don't have time to finish it, what's about creating progress ticket. I also do not see much value in this PR in current state as it doesn't provide jump start for someone else who will pick it up.

**Pull request closed without merging**